Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Author: Peter Michael Green <plugwash@debian.org>
Forwarded: not-needed
Description: debian does not use or ship the embedded C library.
 We link dynamically. So we can skip the C build here.

--- a/build.rs
+++ b/build.rs
@@ -1,11 +1,5 @@
-extern crate cc;
-extern crate pkg_config;
-
-use std::path::PathBuf;
-use std::{env, fs};
-
 fn main() {
-    let mut cfg = cc::Build::new();
+    /*let mut cfg = cc::Build::new();
     let target = env::var("TARGET").unwrap();
     cfg.warnings(false);
 
@@ -43,5 +37,6 @@ fn main() {
     fs::create_dir_all(&include).unwrap();
     fs::copy(src.join("bzlib.h"), dst.join("include/bzlib.h")).unwrap();
     println!("cargo:root={}", dst.display());
-    println!("cargo:include={}", dst.join("include").display());
+    println!("cargo:include={}", dst.join("include").display());*/
+    println!("cargo:rustc-link-lib=bz2");
 }
